iwyu_output.h

```diff
@@ -214,6 +232,9 @@ class IwyuFileInfo {
   // the fwd-decl be removed, even if we don't see any uses of it.
   void AddForwardDeclare(const clang::NamedDecl* forward_declare_decl,
                          bool definitely_keep_fwd_decl);
+    // definitely_keep_fwd_decl tells us that we should never suggest
+  // the fwd-decl be removed, even if we don't see any uses of it.
+  void AddUsingDecl(const clang::UsingDecl* using_decl_decl);
```

Comment:
Actually this was intentional because the type is UsingDecl I felt that using_decl_decl was more descriptive in saying 'this is the declaration of some UsingDecl'.  But I can understand how it looks wrong.  Can you think of another name?